.github/workflows/multinode-inputs.py

```diff
@@ -43,6 +44,16 @@ class Scenario:
 
 
 def main() -> None:
+
+    parser = argparse.ArgumentParser(
+        description='Randomly picks a multinode scenario to execute')
+    parser.add_argument(
+        '--output-summary', '-s',
+        type=argparse.FileType('w', encoding='UTF-8'),
+        default=None,
+        help="Generate a markdown table with selected inputs, can be '-' for stdout")
```

Comment:
Slight clarity improvements
```suggestion
        help="Write a markdown summary table of selected inputs to a file (use '-' to write to stdout)")
```